### PR TITLE
Fix typo in the documentation

### PR DESCRIPTION
**Pull request by Dev-Doggo**
This PR…
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/errors/invalid_array_length/index.md [hunk 1]
```diff
@@ -62,9 +62,9 @@ a.length = a.length - 1;         // set -1 to the length property
 
 let b = new Array(Math.pow(2, 32) - 1);
 b.length = b.length + 1;         // set 2^32 to the length property
-b.length = 2.5;                  // set an integer to the length property
+b.length = 2.5;                  // set the length property to a floating number
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/errors/invalid_array_length/index.md:65]:
```suggestion
b.length = 2.5;                  // set the length property to a floating-point number
```

**Review by wbamberg — DISMISSED** [review]
Thanks! I had a couple of suggestions. Also, I'm not sure if it was intentional not to change lines 61 and 64 as well?

**Comment by Dev-Doggo** [comment]
Lines 61 and 64 were indeed not edited on purpose, as far as I can tell they are accurate?
Also for your suggested changes, I edited them to `floating numbers` as that's what the document refers to them as further up (line 12)

**Comment by wbamberg** [comment]
> Lines 61 and 64 were indeed not edited on purpose, as far as I can tell they are accurate?

In this PR you've reversed:

> // set an integer to the length property

to:

> // set the length property to a floating number

....I assumed because you think "set the $property to $value" is more correct. But line 61 has the order you reversed: "set -1 to the length property". So I'm not sure why 61 is correct and the lines you changed are not.

> I edited them to floating numbers as that's what the document refers to them as further up (line 12)

"floating-point number" is surely a much more common way to refer to these things.

**Review by Josh-Cena — APPROVED** [review]
I've applied the changes myself, because I agree with @wbamberg—specifically, "set [property] to [value]" sounds much, much better than "set [value] to the [property] property".

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**